hugo/assets/sass/blocks/_terminal.scss

```diff
@@ -47,6 +47,9 @@ pre.terminal {
   .information {
     color: $terminal-light-blue;
   }
+  .cmd {
```

Comment:
what's this? I don't see it used anywhere

Reply:
is it different from `command` ?

Reply:
yeah, it is, but it might be ambiguous.

`command` is for wrapping the command that we're showing being typed and run in the terminal, eg `fk status`. It prepends that command with a green prompt `>`.

`cmd` is for making things blue, like we do in fk, when we're suggesting they are 'commands' the user can run.

You're right, I don't appear to actually use it in this PR. I think that's cos I deleted the output of `fk status`

We could drop this `cmd` class, but I think it _will_ be helpful in the future.

Make sense?

Reply:
Riiiight. Let's drop it for now

Reply:
Dropped!